libp2p/Host.cpp

```diff
@@ -24,6 +24,7 @@
 #include <chrono>
 #include <thread>
 #include <mutex>
+#include <boost/asio.hpp>
```

Comment:
answer is in commit message:

> Avoid boost include order issue on windows: …
http://stackoverflow.com/questions/9750344/boostasio-winsock-and-winsock-2-compatibility-issue
> http://stackoverflow.com/questions/9750344/boostasio-winsock-and-winsock-2-compatibility-issue/16288859#16288859
> 
> Make sure boost/asio.hpp is included before windows.h.